src/pages/proposal/detail/VoteOptions.tsx

```
@@ -117,97 +135,136 @@ const VoteOptions = ({
+            />
+          );
+          break;
+        }
+        case ProposalTypes.General: {
+          label = (
+            <Stack direction="row" spacing={1}>
+              {opt.value ? <CheckCircle /> : <Cancel />}
+              <Typography
+                variant="body2"
+                fontWeight={600}
+                dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)